### DISCUSSION TITLE
Change size and dpi of plot in SolaraViz

### DISCUSSION THREAD

**Discussion by gkutlerdodd** [mesa/mesa#2517] [Q&A] [answered]
Hello! I was wondering whether it is possible to increase the size and dpi of a plot on the page when using make_space_component with SolaraViz? Since the function you give to post_process acts on an Axes object, I don't see how you would increase the plot size using this. Thanks for the help!

**Comment by quaquel** [comment, marked as answer]
Good question and no idea. 

You can get the figure from the `Axes` instance (i.e., `ax.fig`), see [matplotlib.Axes](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.html#matplotlib.axes.Axes). Once you have the figure instance, you have access to all methods that are defined on the figure. See [matplotlib.Figure](https://matplotlib.org/stable/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure). There are a `set_size_inches` and `set_dpi` method, so it should be possible to do this.

**Reply by gkutlerdodd** [reply]
Thank you, this worked for changing the dpi! `set_size_inches` did not increase the size of the plot on the page, it just made the agents smaller in the same sized plot, so it looks like the plot size is controlled by SolaraViz and not the plot object itself. Any ideas for this?

**Reply by quaquel** [reply]
Strange: in the [epstein example](https://mesa.readthedocs.io/stable/examples/advanced/epstein_civil_violence.html#app) changing the size does work on my machine.

**Reply by gkutlerdodd** [reply]
It was a typo on my part, I got it to work. Thanks for all the help!

**Reply by EwoutH** [reply]
Thanks for figuring this out!

I think it might be nice to document this somewhere, @gkutlerdodd have a location where you would find this logical?

**Reply by quaquel** [reply]
the simplest would be to add it to the docs for SolaraViz (which don't render yet because of #2451)
